.github/workflows/beaker.yml

```diff
@@ -175,6 +177,10 @@ jobs:
           bundler-cache: true
           cache-version: ${{ inputs.cache-version }}
           working-directory: ${{ inputs.working-directory }}
+      - name: Final vagrant setup
+        if: ${{ inputs.beaker_hypervisor == 'vagrant_libvirt' }}
+        run: |
+          vagrant plugin install vagrant-libvirt
```

Comment:
There's no longer a package for this? That's too bad

Reply:
No longer a package, and vagrant we have to pull from hashi, the plus side is we get newer packages!